### PR TITLE
Expansion panel styles

### PR DESCRIPTION
**Pull request by inikolova**
Part of: https://github.com/telerik/kendo-themes/issues/1950

### CODE REVIEW

**Review by silviyaboteva — COMMENTED** [review]
Great work @inikolova! I have only 2 concerns:
 - the focus state in the Default and Bootstrap themes designs is 1px border, but you have used 2px and 3px inset box-shadow. What is the correct one?
 - why only the first expansion panel doesn't have top padding? It looks strange to me.

**Comment by inikolova** [comment]
@silviyaboteva Silve, below are my comments to the points:

- I discussed with @nedevp in the chat and we used the same focus state like in Panelbar/Menu etc. I will ping him to update the designs;
- This is by design; the main reason is that the expansion panel can be used as a single panel.

**Review by silviyaboteva — APPROVED** [review]
💯

**Review by joneff — APPROVED** [review]
I have one minor comment regarding the background of the tests, but that can be targeted at a later stage

File: packages/default/scss/expansion-panel/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            padding: $expander-content-padding-y $expander-content-padding-x;
+            overflow: auto;
+        }
+
+        // Multiple expansion panels immediately one after another
+        +.k-expander {
+            .k-expander-header {
+                padding-top: $expander-padding-y;
+            }
+        }
```
Comment (PreslavKozovski) [inline packages/default/scss/expansion-panel/_layout.scss:56]:
```suggestion
        + .k-expander .k-expander-header {
            padding-top: $expander-padding-y;
        }
```

File: packages/material/scss/expansion-panel/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+$expander-header-focused-bg: rgba( black, .12 ) !default;
+
+.k-expander {
+    .k-expander-header {
+        &.k-state-focused {
+            background-color: $expander-header-focused-bg;
+        }
+    }
+}
```
Comment (PreslavKozovski) [inline packages/material/scss/expansion-panel/_theme.scss:12]:
```suggestion
.k-expander .k-expander-header .k-state-focused {
    background-color: $expander-header-focused-bg;
}
```

File: packages/material/scss/expansion-panel/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+@import "~@progress/kendo-theme-default/scss/expansion-panel/_theme.scss";
```
Comment (PreslavKozovski) [inline packages/material/scss/expansion-panel/_theme.scss:1]: Wrap the Material theme in a module

File: packages/material/scss/expansion-panel/_theme.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+
+$expander-header-focused-bg: rgba( black, .12 ) !default;
```
Comment (PreslavKozovski) [inline packages/material/scss/expansion-panel/_theme.scss:4]: Move this to the `_variables.scss` file

File: tests/visual/expansion-panel.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Panelbar</title>
```
Comment (PreslavKozovski) [inline tests/visual/expansion-panel.html:4]:
```suggestion
    <title>Expansion Panel</title>
```

File: tests/visual/expansion-panel.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            /*grid-template-columns: repeat(3, 1fr);*/
+            align-items: start;
```
Comment (PreslavKozovski) [inline tests/visual/expansion-panel.html:13]:
```suggestion
```

**Review by PreslavKozovski — CHANGES_REQUESTED** [review]

File: packages/material/scss/expansion-panel/_theme.scss (hunk 4 above)
Comment (inikolova) [reply packages/material/scss/expansion-panel/_theme.scss:4]: Header focused background should be applied only in the Material theme, therefore I added the variable here. Do you think we should add this variable in the Default and Bootstrap themes too?

File: packages/default/scss/expansion-panel/_layout.scss (hunk 1 above)
Comment (inikolova) [reply packages/default/scss/expansion-panel/_layout.scss:56]: We usually use nesting for the selectors. What's the reason for this change

File: packages/material/scss/expansion-panel/_theme.scss (hunk 2 above)
Comment (inikolova) [reply packages/material/scss/expansion-panel/_theme.scss:12]: Same question as above :)

File: packages/default/scss/expansion-panel/_layout.scss (hunk 1 above)
Comment (PreslavKozovski) [reply packages/default/scss/expansion-panel/_layout.scss:56]: Indeed, we are using nested selectors for easier implementation. However, nesting a selector slowers the build and it is harder to read/find in code when only a single declaration is used.

We can add this as a question to the agenda of our tomorrow's meeting, and discuss this topic with other FEs. After that, we can add our decision to the following wiki - https://github.com/telerik/kendo-themes/wiki/Code-Styleguide

File: packages/material/scss/expansion-panel/_theme.scss (hunk 4 above)
Comment (PreslavKozovski) [reply packages/material/scss/expansion-panel/_theme.scss]: I am the person who started the initiative to make all of our theme `variables` similar, however, in some situations, I believe this makes no sense.

I do believe that this is one of them. For example, if we make all the `_variables` files identical, and we import the "Default" layout/theme to the other themes - we no longer have a different theme but a swatch.

With the above said, I believe that it is okay to have a different number of variables for the different themes when it makes sense.

We can also add this as a topic for our meeting.

Comment (silviyaboteva) [reply packages/material/scss/expansion-panel/_theme.scss]: Can we have the same variable in the Default and Bootstrap themes, but with `null` value?

Comment (inikolova) [reply packages/material/scss/expansion-panel/_theme.scss]: Yes, we could, but do we need them? We will discuss this case in our meeting

**Review by PreslavKozovski — APPROVED** [review]

**Review by joneff — APPROVED** [review]

**Pull request merged**